src/main/java/com/demo/classroom/Controller/AuthController.java

```diff
@@ -0,0 +1,39 @@
+package com.demo.classroom.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.demo.classroom.DTO.RegistrationDTO;
+import com.demo.classroom.Service.RegistrationService;
+import com.demo.classroom.Utility.ErrorMessages;
+
+import jakarta.validation.Valid;
+
+@RestController
+@RequestMapping("/req")
+public class AuthController {
```

Comment:
Why should we start with `/req` can't me implement it in a way that i can access the signup with endpoint like `/signup` ?

Reply:
yes we can. Resolved.